doc/radosgw/multisite.rst

```diff
@@ -528,6 +528,15 @@ running the following commands on the object gateway host:
    systemctl start ceph-radosgw@rgw.`hostname -s`
    systemctl enable ceph-radosgw@rgw.`hostname -s`
 
+If you have used the ``radosgw-admin`` command, you will not be able to use
```

Comment:
I'm skeptical of this wording.  The above steps include a pool deletion that is qualified as only to be run if there is no data, implying that the other steps are be run regardless.  I don't understand now `radosgw-admin` implies that there are no services.  I suggest that this be run by the RGW leads.

Oh and above the mention of `civetweb` is outdated.

Reply:
i think this is pointing at the difference between package vs containerized deployments, but confuses `radosgw-admin` for `cephadm`